### ISSUE TITLE
IR LEDs for night mode

### ISSUE THREAD

**Issue by jymbob** [ghoost82/mijia-720p-hack#15]
Successfully using v0.95 of this hack with the Chuangmi camera, which is the same chipset but without the motor. Many thanks, first of all.

An issue I'm seeing is that while the auto day/night mode is working, switching to night mode doesn't switch on the IR LEDs.

I believe this is because of the issue mentioned here: https://github.com/cck56/mijia-720p-ctrl/issues/1#issuecomment-389120771

If I ssh into the camera and `killall ir_sample` I can then run `mijia_ctrl IRLED 255`

Any idea how to keep auto day/night while also being able to control the IR LEDs?

**Comment by max5962** [comment]
@ghoost82 any news  ?: )

**Comment by jaypikay** [comment]
@jymbob You have to run 3 commands to enable NV.

NV ON (Max IR LED Brightness):

    DAYNIGHT,1
    IRCUT,0
    IRLED,255

NV OFF:

    DAYNIGHT,0
    IRCUT,1
    IRLED,0

**Comment by jymbob** [comment]
Yes, I'm aware of that. The issue is that this should happen as part of irsample but it doesn't. There's a fork that's working towards a more comprehensive set of functions

**Comment by max5962** [comment]
Can you send me the fork url  ?: )

**Comment by tomsteenbakkers** [comment]
Is there a way to execute the `killall ir_sample` and `mijia_ctrl IRLED 255` at startup?

**Comment by fliphess** [comment]
@max5962 @tomsteenbakkers  Try [my fork](https://github.com/fliphess/chuangmi-720p-hack) for use with the chuangmi.

**Comment by tomsteenbakkers** [comment]
just installed the hack. First I downgrade the firmware. This was successful.

Then I copied all the files on the SD and modified config.cfg. Changes the passwords and the Wifi settings. But when booting the camera with this SD, I can not access the web interface. IP number is assigned, but the website is not working. Any ideas?

**Comment by tomsteenbakkers** [comment]
> @max5962 @tomsteenbakkers Try [my fork](https://github.com/fliphess/chuangmi-720p-hack) for use with the chuangmi.

@fliphess just installed the hack. First I downgrade the firmware. This was successful.

Then I copied all the files on the SD and modified config.cfg. Changes the passwords and the Wifi settings. But when booting the camera with this SD, I can not access the web interface. IP number is assigned, but the website is not working. Any ideas?

How can I check which firmware is on the camera. I want to check if the downgrade was really successful. The file name of the recovery firmware files was changed, but I doubt if it was successful.

**Comment by fliphess** [comment]
Hey @tomsteenbakkers I created a new release this weekend. There were some issues in the lighttpd.conf in master due to incorrect chars as a result of improper copy pasta.

If you use the releases rather than your own build from master i expect it to work now. 
Can you test If it works and otherwise create an issue in my repo so I can have a look at it? Thanks! :)

**Comment by tomsteenbakkers** [comment]
@fliphess I installed the new version, but no changes with the previous version. The camera is booting and it is also recording, they are stored in the sd-card. When the blue light is on, I can't connect using SSH with the camera. Also browsing to the IP nummer has no result. 

Is there a log-file where I can see where it goes wrong.

**Comment by gr3k** [comment]
@jymbob solution is to rename/remove binary: mijia_ctrl from mijia-720p-hack/bin/  .It cause the problem. (You need to reboot the camera after that) 
After that S99auto_night_mode script working without any issue. 
Its more. When I had mija_ctrl binary my camera after the reboot couldn't remember last position and I was force to set it up manually.
After removing that binary this problem was solved. 
So currently I have Auto night mode ,hurray
